products/micronaut.md

```diff
@@ -17,6 +17,13 @@ auto:
 -   git: https://github.com/micronaut-projects/micronaut-core.git
 
 releases:
+-   releaseCycle: "4"
+    releaseDate: 2023-07-11
+    eol: false
+    support: true
+    latest: "4.0.0"
+    latestReleaseDate: 2023-07-11
+
 -   releaseCycle: "3"
     releaseDate: 2021-08-18
     eol: false
```

Comment:
support for v3 should be `2023-07-11` (v4 releaseDate)

Reply:
Then we assume that they move v3 from "Active Development" to "Active Maintenance", but this is okay for me, going to update the line.